### PR TITLE
feat: new Microsoft Teams alert channel resource

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Microsoft Teams alert channel resource

Signed-off-by: Darren Murray <darren.murray@lacework.net>

### CODE REVIEW

**Review by scottford-lw — APPROVED** [review]
🤘

File: website/docs/r/alert_channel_microsoft_teams.html.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_microsoft_teams"
+description: |-
+  Create and manage Microsoft Teams Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_microsoft_teams
+
+You can configure Lacework to forward alerts to a Microsoft Teams channel through an incoming webhook.
```
Comment (afiune) [inline website/docs/r/alert_channel_microsoft_teams.html.markdown:11]: Could we add a link to https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook

**Review by afiune — APPROVED** [review]
let's make sure to sync the proper dependencies once the PR over at our Go-SDK is merged.

tenor-89748228

File: website/docs/r/alert_channel_microsoft_teams.html.markdown (hunk 1 above)
Comment (dmurray-lacework) [reply website/docs/r/alert_channel_microsoft_teams.html.markdown:11]: Added!

**Pull request merged**